packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -0,0 +1,263 @@
+import _, { messages } from 'intl'
+import ActionButton from 'action-button'
+import decorate from 'apply-decorators'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import Wizard, { Section } from 'wizard'
+import { map } from 'lodash'
+import { Toggle } from 'form'
+import { connectStore } from 'utils'
+import { injectIntl } from 'react-intl'
+import { linkState } from 'reaclette-utils'
+import { Container, Row, Col } from 'grid'
+import { provideState, injectState } from 'reaclette'
+import { SelectPool, SelectPif } from 'select-objects'
+import { getBondModes, createBondedNetwork, createNetwork } from 'xo'
+import { createGetObject, getIsPoolAdmin } from 'selectors'
+
+import styles from './index.css'
+import Page from '../../page'
+
+const getInitialState = () => ({
+  bondedNetwork: false,
+  bondMode: undefined,
+  bondModes: undefined,
+  description: '',
+  mtu: '',
+  name: '',
+  pif: undefined,
+  pifs: undefined,
+  vlan: '',
+})
+
+const NewNetwork = decorate([
+  connectStore(() => ({
+    isPoolAdmin: getIsPoolAdmin,
+    pool: createGetObject((_, props) => props.location.query.pool),
+  })),
+  injectIntl,
+  provideState({
+    initialState: getInitialState,
+    effects: {
+      linkState,
+      initialize: async () => {
+        let bondModes
+        await getBondModes().then(modes => {
+          bondModes = modes
+        })
```

Comment:
Avoid mixing `async/await` and `.then`: `const bondModes = await getBondModes()`